test/e2e/tests/dapp-interactions/block-explorer.spec.js

```diff
@@ -82,30 +79,30 @@ describe('Block Explorer', function () {
         await driver.clickElement(
           '[data-testid="account-overview__asset-tab"]',
         );
-        const [, tst] = await driver.findElements(
-          '[data-testid="multichain-token-list-button"]',
-        );
-        await tst.click();
```

Comment:
the test failed at this point `TypeError: Cannot read properties of undefined (reading 'click')`. We should use the `clickElement` from our driver instead, so this will make sure that the element is clickable